### ISSUE TITLE
Ability to specify release ID instead of tag

### ISSUE THREAD

**Issue by kevcube** [robinraju/release-downloader#551]
**Is your feature request related to a problem? Please describe.**
I have a workflow that outputs a release ID and for higher precision I'd like to use this instead of the tag to identify a release

**Describe the solution you'd like**
An input for release ID

**Describe alternatives you've considered**
I can just pass the tag

**Additional context**
I can probably PR this but haven't looked at the code yet.

**Comment by robinraju** [comment]
@kevcube If you prefer submitting a PR, you can add `release_id` to the outputs by making the following changes

add field `id` to https://github.com/robinraju/release-downloader/blob/ca4614ff79f1fbac45b5572e6fa1018e47a7b798/src/gh-api.ts#L6-L12

And set the additional output `release_id` using the `id` field along https://github.com/robinraju/release-downloader/blob/ca4614ff79f1fbac45b5572e6fa1018e47a7b798/src/release-downloader.ts#L37

Here is a sample JSON where we get release `Id` from: https://github.com/robinraju/release-downloader/blob/ca4614ff79f1fbac45b5572e6fa1018e47a7b798/__tests__/resource/1-release-latest.json#L6

**Comment by kevcube** [comment]
@robinraju I'm looking to add release ID to the inputs of this workflow, not outputs. working on that now.

**Comment by kevcube** [comment]
see #563

**Comment by robinraju** [comment]
@kevcube I just merged #563 you may test it by using this action from the merged commit as follows until the next version is released.

```
- uses: robinraju/release-downloader@d64632db82bd601ff8d4822ad9a50dd7095b1a2f
  with:
    released: "123123"
    fileName: "foo.zip"
```
